userguide/install.rst

```diff
@@ -120,16 +120,18 @@ Meizu devices are pretty much stuck on Flyme. While the MX4 can be flashed succe
 
     BE VERY CAREFUL! You are responsible for your own actions!
 
-1. Disconnect all devices and non-essential peripherals from your PC. Charge your device on a wall-charger (not your PC) to at least 40 percent
-2. Download the Ubuntu Touch ROM for your device:
+1. Disconnect all devices and non-essential peripherals from your PC. Charge your device on a wall-charger (not your PC) to at least 40 percent.
+2. Download the Ubuntu Touch ROM for your device and extract the zip files:
 
     - `Bq E4.5 <http://cdimage.ubports.com/ubports-installer/krillin/e45.zip>`_ (*krillin*)
     - `Bq E5 HD <https://storage.googleapis.com/otas/2014/Smartphones/Aquaris_E5_HD/Ubuntu/VEGETA01A-S11A_BQ_L100EN_2017_170207.zip>`_ (*vegetahd*)
     - `Bq M10 HD <https://storage.googleapis.com/otas/2015/Tablets/Freezer/Ubuntu/OTA_13_FW_version/cooler-image-stable-6.zip>`_ (*cooler*)
     - `Bq M10 FHD <https://storage.googleapis.com/otas/2015/Tablets/Freezer%20FHD/Ubuntu/OTA_15/frieza-image-stable-8.zip>`_ (*frieza*)
     - `Meizu MX4 <http://people.canonical.com/~alextu/tangxi/684.zip>`_ (*arale*)
 
-3. Download `SP flash tool <https://storage.googleapis.com/otas/2015/Tablets/Freezer%20FHD/Ubuntu/SP_Flash_Tool_exe_Linux_v5.1612.00.100.zip>`_ for Linux.
+3. Download the latest version of `SP flash tool <https://spflashtool.com/download/>`_ (*aka MTK flash tool*).
+
+4. Remove the ``modemmanager`` package as `the one installed is not compatible with SP flash tool <https://web.archive.org/web/20180118130643/http://www.mibqyyo.com/en-articles/2015/09/16/ubuntu-android-installation-process-for-bq-aquaris-e4-5-and-e5/>`_.
 
 On Ubuntu 17.10, there are issues with flash_tool loading the shared library 'libpng12', so this can be used as a workaround:
```

Comment:
it would be nice to format this consistently with the modemmanager tip. 

ie if the modemmanager is step 4, then I think this one should be step 5, not some strangely aligned paragraph interupting the enumeration. 

if we would turn modemmanager into a note (my preference) or a sub bullet under the "run it" step, then I think the libpng tip should go there as well